src/parsec_service/key_info_managers.md

```diff
@@ -0,0 +1,13 @@
+# Key Info Managers
+
+The key info stored by the key info managers consist in a structure composed of: * an `id`, which is
+the reference to a key in the specific provider * `attributes`, which are the attributes of the key
```

Comment:
Is that supposed to be/look like a list?

Reply:
Oops! Forgot a space, nice catch